src/components/link/link.style.js

```diff
@@ -0,0 +1,40 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../style/themes/base';
+
+const LinkStyle = styled.div`
+  a {
+    font-size: 14px;
+    text-decoration: underline;
+    color: ${({ theme }) => theme.colors.primary} !important;
+
+    &:hover {
+      cursor: pointer;
+      color: ${({ theme }) => theme.colors.secondary} !important;
```

Comment:
you have `!important` twice for the same property

Reply:
This is for hover example